File: src/basic_training/Day24_1.java
package basic_training;

public class Day24_1 {
	public int solution(String[] order) {
		int price = 0;
		for(String menu : order) {
			price += (menu.contains("americano") || menu.contains("anything")) ? 4500 : 5000;
		}
		return price;
	}
}

File: bin/basic_training/Day24_1.js
function solution(order) {
	return order.reduce((acc, cur) => acc + (cur.includes("americano") ? 4500 : 5000)
	, 0)
}


// 참고할만한 번외 문제풀이
// function solution(order) {
// 	return order.reduce((acc, cur) => acc + (cur.includes("latte") ? 5000 : 4500), 0)
// }